renderer/util/minor.ts

```diff
@@ -1,4 +1,35 @@
-export function convertToGbFromPetabytes(value: number): number {
-  const gigabytes = value * 1e9; // 1 petabyte = 1e9 gigabytes
-  return parseFloat(gigabytes.toFixed(2));
+import { ValueWithMeasureValue } from "../types/Charts";
+const size_mv_list = ["B", "KB", "MB", "GB", "TB", "PB"];
+
+// rough port from
+// https://github.com/virdpool/ar_miner_ui_playground/blob/e28a8034bd367a35fe40878c90c606e456b2346f/electron_app/htdocs/util/fmt.coffee#L6
+// TODO refactor
+export function fmt_size(orig_value: number): ValueWithMeasureValue {
```

Comment:
I think the use of camelCase is very well established as the casing for functions in javascript

Reply:
Fixed and also removed duplicate in neighbor file

Reply:
looks good now